.pipelines/verifyDepsJsonLibraryVersions.ps1

```diff
@@ -15,7 +15,8 @@ Param(
 $referencedFileVersionsPerDll = @{}
 $totalFailures = 0
 
-Get-ChildItem $targetDir -Recurse -Filter *.deps.json | ForEach-Object {
+Get-ChildItem $targetDir -Recurse -Filter *.deps.json -Exclude UITests-FancyZones* | ForEach-Object {
+    # Temporarily exclude FancyZones UI tests because of Appium.WebDriver dependencies
```

Comment:
I suppose these will only get built in the tests folder, so there's no chance of binary collisions, right?

Reply:
That's right, the output is in the tests folder.